cloudmarker/test/test_filestore.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for filestore plugins."""
+
+
+import os
+import pathlib
+import shutil
+import unittest
+
+from cloudmarker.stores import filestore
+
+
+class FileStoreTest(unittest.TestCase):
+    """Tests for filestore plugins."""
+
+    @classmethod
+    def write_to_file(cls, filename):
```

Comment:
Since the `cls` argument is never used, it is technically not a class method. It does not work on the class. This could be a static method (which in Python is just syntactic sugar). A much simpler and obvious way to write this function would be as a module level function.

I almost always prefer a module level function to a static method function.

Reply:
The [`tearDownClass`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDownClass) method is related to the `unittest` class and is marked to be called after the completion of all the tests in that particular class. The library docs say that it should be marked as a `@classmethod`.